content/blog/18-19/g2_blogpost.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "A Gateway to collaboration"
+created_at: 26-02-2019
+description: Het ProjectenProject
+author: Arne Bertrand
+---
+> Ik wil meedoen met een zeus project, hoe doe?
+
+Het is een vraag die enorm vaak voorkomt en verbazend moeilijk te 
+beantwoorden is. De keuze van het project op zich is vaak al lastig. 
+Welke technologie wil ik gebruiken? Welke projecten zijn nog relevant?
+Welk project heeft het meeste hulp nodig? Het zijn stuk voor stuk aspecten
+die meespelen en elkaar vaak tegenspreken.
+
+Eens een project gekozen, komt er vaak nog een hele boterham setup bij kijken
+die, we moeten het eerlijk toegeven, niet altijd goed gedocumenteerd is.
+Eens dan eindelijk de setup voltooid is, komt nog het moeilijkste deel:
+het selecteren van een issue die:
+
+- relevant is
+- binnen je niveau ligt
+- enigszins doenbaar is in de beperkte tijd die je kan vrijmaken
+
+
+## G2 - Gateway
+
+De oplossing? NOG een project natuurlijk! G2 wordt onze eigenste
+project-management hub. Het zal projectleiders de tools aanbieden om issues
+en branches op een logische manier te structureren. Projecten worden 
+recursief onderverdeeld in subprojecten, die elk op zich een logisch
+samenhangende collectie van issues en branches zijn. 
+
+Ja, dit kan nu ook al op zekere hoogte met zaken zoals tags op git(hub|lab). 
+De meerwaarde zit dan ook vooral in de andere richting.
+
+Nieuwe leden kunnen op een eenvoudige en gecentraliseerde manier de lopende
+projecten verkennen. Hierbij moeten ze niet steeds de repos doorspitten,
+maar kunnen ze zoeken op talen, frameworks en "soort" projecten (backend, 
+frontend, etc). 
+
+Leden met al enige pull requests achter de kiezen kunnen na een hiatus
+zich makkelijk heroriënteren in hun favoriete projecten, en een 
+subproject uitkiezen dat hun aanspreekt.
```

Comment:
hun -> hen

Reply:
waarom is nederlands

Reply:
Welkom bij Niko's taalweetjes!

----

Uit https://onzetaal.nl/taaladvies/hun-hen:
> aanspreken: dat spreekt hen wel aan

**Goed en wel, maar waarom?**
We gaan op amateurtaalonderzoek. De regels staan ook op dezelfde pagina, maar ook uit https://www.taaltelefoon.be/hen-hun-ze:
> Voor een lijdend voorwerp gebruiken we hen.

**Ja maar betreft het hier een lijdend voorwerp?**

In de _Van Dale Groot Woordenboek Hedendaags Nederlands_ (dat hier bij mij ligt) vinden we:
> **aanspreken** [ov.ww., sprak aan, h aangesproken]

`ov.ww.` staat voor overgankelijk werkwoord.

**Wat is dat nu weer?**
We kijken naar https://taaladvies.net/taal/advies/term/67/overgankelijk_onovergankelijk_werkwoord/
> Een overgankelijk werkwoord is een werkwoord dat in een zin altijd een lijdend voorwerp bij zich kan hebben

Aangezien bij het gebruik van _aanspreken_ het onderwerp hier duidelijk _subproject_ is, lijkt het logisch dat _hen_/_hun_ hier het lijdend voorwerp is.

**Goed goed, maar waarom is er een verschil tussen hen/hun?**
Opnieuw uit https://onzetaal.nl/taaladvies/hun-hen:
> Dat we verschil zouden moeten maken tussen hen en hun in ‘Ik zie hen’, ‘Ik vertel het hun’ en ‘Ik vertel het aan hen’ is een regel die we te danken hebben aan de zeventiende-eeuwse wetenschapper Christiaen van Heule. Deze regel heeft nooit echt geleefd in het dagelijks taalgebruik: hen en hun werden en worden door elkaar gebruikt in dit soort zinnen.

Als we willen weten wat Christiaen bezielde, kunnen doorklikken naar https://www.ernieramaker.nl/schrijfsels.php?tekst=henhun. Omdat dit een hele pagina is, geef ik hier een korte samenvatting:

> In zijn boek uit 1625 beschrijft Christiaen de grammatica der Nederlandse taal (met als prachtige titel _De Nederduytsche Grammatica ofte Spraec-konst_). Hij doet dit door in essentie het Latijnse naamvallensysteem op het Nederlands te forceren. Hij beschrijft met andere woorden de taal niet, maar vindt ze ter plekke uit. Waarom doet hij dat? Omdat het Latijn in die tijd een zeer hoog aanzien genoot; ervoor zorgen dat het Nederlands over een gelijkaardig systeem beschikte was wenselijk.

Tot slot nog opmerken dat door recente ontwikkelingen (recent als in 1997) in de _Algemene Nederlandse Spraakkunst_) vermeld wordt dat _hen_ gebruiken in plaats van _hun_ niet als een fout kan beschouwd worden, en dat gecombineerd met het feit dat _hen_ stilistisch hoger gewaardeerd wordt, het aanvaardbaar is om consequent overal _hen_ te gebruiken.

----
Dit was het voor Niko's taalweetje.